packages/react/src/core/tokens.ts

```diff
@@ -128,6 +128,18 @@ const zIndex = {
 
 export type ZIndex = keyof typeof zIndex;
 
+/** A set of predefined box-shadows. */
+const shadow = {
```

Comment:
Not sure if we need these documentation here – we don't for other design tokens. This content already exists on the tokens page.